core/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java

```diff
@@ -889,7 +918,11 @@ public void testSycnedFlushSurvivesEngineRestart() throws IOException {
         } else {
             engine.flushAndClose();
         }
-        engine = new InternalEngine(config, randomBoolean());
+        engine = new InternalEngine(copy(config, EngineConfig.OpenMode.OPEN_INDEX_AND_TRANSLOG));
```

Comment:
this is different than what it was - before if we had false, we'll ignore the local translog and commit a checkpoint. no we just ignore the recovery, but do not commit.  I think we should keep old behavior and use OPEN_INDEX_CREATE_TRANSLOG randomly.